dev/package.json

```diff
@@ -3,5 +3,8 @@
     "eslint": "^7.25.0",
     "ansi-regex": "^5.0.1",
     "minimatch": "^3.1.2"
+  },
+  "resolutions": {
```

Comment:
O.K, `resolutions` is better place we add `"optionator": "^0.9.3"` than `devDependencies`.

Reply:
eh.. is this OK? or will you have something else?

Reply:
Yeah, I think this is OK.

Reply:
No, all word-wrap are from @aashutoshrathi/word-wrap": "^1.2.3",



![image](https://github.com/apache/spark/assets/47577197/66adb31d-85f3-4d35-8e69-2398d5770ea6)

but we need 1.2.4
![image](https://github.com/apache/spark/assets/47577197/34ed7fbe-3043-436a-b47e-8b76721f07ce)

I think we have to go for 
"resolutions": {
  "word-wrap": "npm:@aashutoshrathi/word-wrap@^1.2.4"
},

Reply:
yeah, we need 1.2.4 https://github.com/jonschlinkert/word-wrap/pull/33#issuecomment-1509703360

Reply:
we got  "node_modules/@aashutoshrathi/word-wrap": {
      "version": "1.2.6",

Reply:
We can get `1.2.6` regardless of `^1.2.3` or `^1.2.4`.
Also, aashutoshrathi doesn't provide `1.2.3` so we never get `1.2.3`.
https://www.npmjs.com/package/@aashutoshrathi/word-wrap?activeTab=versions